content/docs/command-reference/remote/modify.md

```diff
@@ -273,38 +273,54 @@ Operational parameters:
   > - [ACL Overview - Permissions](https://docs.aws.amazon.com/AmazonS3/latest/dev/acl-overview.html#permissions)
   > - [Put Object ACL](https://docs.aws.amazon.com/AmazonS3/latest/API/API_PutObjectAcl.html)
 
-S3 remotes can also be configured entirely via environment variables:
+S3 remotes can also be configured entirely via environment variables, e.g.:
 
 ```dvc
 $ export AWS_ACCESS_KEY_ID='mykey'
 $ export AWS_SECRET_ACCESS_KEY='mysecret'
 $ dvc remote add -d myremote s3://mybucket/path
 ```
 
-For more information about the variables DVC supports, please visit
-[boto3 documentation](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html#environment-variable-configuration)
+For more on the supported env vars, please see the
+[boto3 docs](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html#environment-variable-configuration)
```

Comment:
this actually, can stay as part of the `dvc remote add` - since it's not about auth or config, it's more about remote spec and what to expect

Reply:
Oh OK I see you meant the `boto3` permission stuff earlier. You're right about that part being too detailed, removed from `remote add` in 10f177fa.

Reply:
I was not suggesting to remove it :) It has certain value to it - we specify the list of permissions that admins would need to enable for DVC to operate. We might change language a bit, but this information is good overall.